tests/stubdata/output/jats_aj_158_4_139.json

```diff
@@ -6,7 +6,6 @@
     {
       "affiliation": [
         {
-          "affPubID": "ajab3643af1",
```

Comment:
i may have missed what these characters are, but are we sure we don't need to capture them anywhere in the schema?

Reply:
I made a judgement call on this, and I think it might've been different from what I said earlier in the parser design process.  What we're trying to represent in the data model is metadata about the object being described by the record (i.e. the publication).  These are keys internal to the document that link an affiliation string to a datum that references the key.  We're not currently capturing the complete XML in this record, so the "key" in this case isn't useful data in the context of the ingest data model.  You could recover what the key means by parsing the document again, but you could also recover the key at that time too.   My guess with this particular ID is that this is Astronomical Journal, submission ab3643, affiliation 1 -- it's primarily a detail of the editorial process for the data, not part of the data itself.  It's an internal identifier that doesn't hold any intrinsic meaning outside of the editorial process, or in relation to other records that aren't using AJ's editorial process.  It's sort of like the case where we were populating this with "Aff1" or "Aff335" -- there are probably millions of records where the input metadata keys an affiliation datum with "Aff1", but there's no meaning in it outside the metadata file of that particular record.

Reply:
this makes sense to me. I think everything else looks good.